go

```diff
@@ -27,7 +28,62 @@ end
 require 'guides_style_18f'
 
 extend GoScript
-check_ruby_version '2.1.5'
+check_ruby_version '2.2.3'
```

Comment:
Ah, given my note above, I take it this check requires looking for a specific version?  I ask because version 2.3.0 is out now.